main/tests.py

```diff
@@ -1,22 +1,23 @@
 import pytest
+
+from daycare.models import DayCare
 from dogowner.models import DogOwner
 
 
 @pytest.fixture
 def create_dog_owner_user():
-    return DogOwner.create(email='dogowner@address.com',
-                           username='dogOwnerUser01',
-                           password='password123',
-                           dog_name='dog name',
-                           first_name='test',
-                           last_name='user',
-                           phone_number=1234567890,
-                           dog_race='dog race',
-                           dog_picture_url='https://www.google.com/user1.jpg',
-                           dog_age=4,
-                           dog_weight=2,
-                           dog_gender='M'
-                           )
+    return DogOwner.create(email='dogowner@address.com', username='dogOwnerUser01', password='password123',
```

Comment:
Now that we merged the conftest file, for each there is no need to exists any `@pytest.fixture` code.
in your case you can just use the `createdog_owner_user` in any place you want and you should delete it from here

Reply:
Removed `createdog_owner_user` fixture